sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/AADClientRegistrationRepository.java

```diff
@@ -60,4 +61,9 @@ public boolean isClientNeedConsentWhenLogin(String id) {
         ClientRegistration client = findByRegistrationId(id);
         return client != null && isClientNeedConsentWhenLogin(client);
     }
+
+    public static boolean isDefaultClient(ClientRegistration clientRegistration) {
+        return AADWebAppConfiguration.AZURE_CLIENT_REGISTRATION_ID.equals(
```

Comment:
Move `AZURE_CLIENT_REGISTRATION_ID` to `AADClientRegistrationRepository.java`, it's more related to client registration.

Reply:
I'll do ti.